scoring/score.py

```diff
@@ -1,6 +1,13 @@
 import collections
 
-POINTS_PER_TERRITORY = 2
+DEFAULT_POINTS_PER_TERRITORY = 2
+EXTRA_VALUE_TERRITORIES = {
```

Comment:
If I recall correctly we're relying on these extra-points territories not being present in earlier versions of the arena? Please could we have a comment here about that (I realise we can't easily validate it).

Reply:
24954224a466fb4f9e7372d1893a1570d43b94ee